### PR TITLE
Create contributing.md

### PR DESCRIPTION
**Pull request by rahmot**
I created a `contributing.md` file based on this [discussion](https://github.com/bebatut/galaxy_mentor_network/pull/3#discussion_r793332872). Thanks!

### CODE REVIEW

**Comment by bebatut** [comment]
It looks good. 
Can you
- rename it `CONTRIBUTING.md`? There is currently a typo in the name
- add a sentence to give credit to the source? As it is mostly a copy of the one in OLS
Thanks

**Comment by rahmot** [comment]
Thanks, @bebatut! I have modified the file.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+6. Submit a pull request from that branch to the master repository
+7. If you receive feedback, make changes in your local clone and push them to your branch on GitHub: the pull request will update automatically
+8. Pull requests will be merged by the training team members after at least one other person has reviewed the Pull request and approved it.
+
+## What can I do to help the project?
+
+As we move forward with this project, you will find lists of [issues](https://github.com/bebatut/galaxy_mentor_network/issues) to fix and features to implement. Feel
+free to work on them!
+
+**Note:** The texts in this file were copied from [Open Life Science](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) but made a few changes to it.
```
Comment (bebatut) [inline CONTRIBUTING.md:55]:
```suggestion
**Note:** The content of this file was highly inspired from the [Open Life Science `CONTRIBUTING.md` file](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md).
```

**Comment by rahmot** [comment]
Thanks for rephrasing @bebatut.

**Comment by bebatut** [comment]
It looks good. I merge. Thanks

**Comment by rahmot** [comment]
Thanks for merging, I will update the `readme` file now

**Pull request merged**